mteb/models/misc_models.py

```diff
@@ -0,0 +1,1582 @@
+    similarity_fn_name="cosine",
+    use_instructions=None,
+    zero_shot_benchmarks=None,
+    adapted_from=None,
+    superseded_by=None,
+)
+consciousAI__cai_lunaris_text_embeddings = ModelMeta(
+    name="consciousAI/cai-lunaris-text-embeddings",
+    revision="8332c464d13505968ff7a6e2213f36fd8730b4c7",
+    release_date="2023-06-22",
+    languages=None,
+    loader=None,
+    n_parameters=None,
+    memory_usage=None,
+    max_tokens=512,
+    embed_dim=1024,
+    license="apache-2.0",
+    open_weights=True,
+    public_training_data=False,
+    public_training_code=None,
+    framework=["PyTorch"],
+    reference="https://huggingface.co/consciousAI/cai-lunaris-text-embeddings",
+    similarity_fn_name="cosine",
+    use_instructions=None,
+    zero_shot_benchmarks=None,
+    adapted_from=None,
+    superseded_by=None,
+)
+consciousAI__cai_stellaris_text_embeddings = ModelMeta(
+    name="consciousAI/cai-stellaris-text-embeddings",
+    revision="c000ec4b29588daf0f4a0b2ad4e72ee807d8efc0",
+    release_date="2023-06-23",
+    languages=None,
+    loader=None,
+    n_parameters=None,
+    memory_usage=None,
+    max_tokens=514,
+    embed_dim=768,
+    license=None,
+    open_weights=True,
+    public_training_data=False,
+    public_training_code=None,
+    framework=["PyTorch"],
+    reference="https://huggingface.co/consciousAI/cai-stellaris-text-embeddings",
+    similarity_fn_name="cosine",
+    use_instructions=None,
+    zero_shot_benchmarks=None,
+    adapted_from=None,
+    superseded_by=None,
+)
+manu__bge_m3_custom_fr = ModelMeta(
+    name="manu/bge-m3-custom-fr",
+    revision="ed3ef88678ba83ddf4c0fab71a93cb90d89a9078",
+    release_date="2024-04-11",
+    languages=None,
+    loader=None,
+    n_parameters=567754752,
+    memory_usage=None,
+    max_tokens=8194,
+    embed_dim=1024,
+    license=None,
+    open_weights=True,
+    public_training_data=False,
+    public_training_code=None,
+    framework=["PyTorch", "Sentence Transformers"],
+    reference="https://huggingface.co/manu/bge-m3-custom-fr",
+    similarity_fn_name="cosine",
+    use_instructions=None,
+    zero_shot_benchmarks=None,
+    adapted_from=None,
+    superseded_by=None,
+)
+manu__sentence_croissant_alpha_v0_2 = ModelMeta(
+    name="manu/sentence_croissant_alpha_v0.2",
+    revision="4610b8cea65d7dd59e0b04af50753933fe5b29b2",
+    release_date="2024-03-15",
+    languages=None,
+    loader=None,
+    n_parameters=1279887360,
+    memory_usage=None,
+    max_tokens=2048,
+    embed_dim=2048,
+    license=None,
+    open_weights=True,
+    public_training_data=False,
+    public_training_code=None,
+    framework=["PyTorch", "Sentence Transformers"],
+    reference="https://huggingface.co/manu/sentence_croissant_alpha_v0.2",
+    similarity_fn_name="cosine",
+    use_instructions=None,
+    zero_shot_benchmarks=None,
+    adapted_from=None,
+    superseded_by=None,
```

Comment:
Might be superseded by manu/sentence_croissant_alpha_v0.3?

Reply:
Could be. It would be hard to implement logic for this tho. I will fix it manually